pkg/lib/keptn/keptn_base.go

```diff
@@ -157,22 +157,20 @@ func (k *KeptnBase) GetKeptnResource(resource string) (string, error) {
 
 	// return Nil in case resource couldn't be retrieved
 	if err != nil || requestedResource.ResourceContent == "" {
-		fmt.Printf("KeptnBase Resource not found: %s - %s", resource, err)
-		return "", err
+		return nil, fmt.Errorf("resource not found: %s - %s", resource, err)
 	}
 
-	// ToDo: Verify why we need TrimSuffix here
-	return strings.TrimSuffix(requestedResource.ResourceContent, "\n"), nil
+	return []byte(requestedResource.ResourceContent), nil
 }
 
 /**
  * Retrieves a resource (=file) from the local file system. Basically checks if the file is available and if so returns it
  */
-func _getKeptnResourceFromLocal(resource string) (string, error) {
+func _getKeptnResourceFromLocal(resource string) ([]byte, error) {
```

Comment:
can we remove the underscore in `_getKeptnResourceFromLocal(...)`

Reply:
sure